src/databricks/labs/ucx/azure/access.py

```diff
@@ -54,6 +56,50 @@ def __init__(
             "Storage Blob Data Owner": Privilege.WRITE_FILES,
             "Storage Blob Data Reader": Privilege.READ_FILES,
         }
+        self._permission_levels = {
+            "Microsoft.Storage/storageAccounts/blobServices/containers/write": Privilege.WRITE_FILES,
+            "Microsoft.Storage/storageAccounts/blobServices/containers/blobs/write": Privilege.WRITE_FILES,
+            "Microsoft.Storage/storageAccounts/blobServices/containers/read": Privilege.READ_FILES,
+            "Microsoft.Storage/storageAccounts/blobServices/containers/blobs/read": Privilege.READ_FILES,
+        }
+
+    def _get_permission_level(self, permission_to_match: str) -> Privilege | None:
+        # If string contains '*', construct a pattern with wildcard
+        if '*' in permission_to_match:
+            parts = re.escape(permission_to_match).split(r'\*')
+            pattern = '^' + parts[0] + '.*' + parts[1] + '$'
```

Comment:
this one feels fragile to me, what if there are multiple * to match?

Reply:
Is the regex needed? Would a exact match not suffice?

Reply:
@nkvuong Please let me know if this makes sense, or if some changes should be made.